.travis.yml

```diff
@@ -44,6 +44,22 @@ jobs:
         - julia --project -e 'using Pkg; Pkg.activate("docs"); Pkg.instantiate(); Pkg.develop(PackageSpec(path = pwd()))'
         - julia --project=docs --color=yes docs/make.jl
       after_success: skip
+  include:
+    - name: TOML (Julia 1.0)
+      stage: test
+      os: linux
+      julia: 1.0
+      script:
+        - julia --project=ext/TOML -e 'using Pkg; Pkg.test()'
+      after_success: skip
+  include:
```

Comment:
Looks like this section overwrites the one above. Trying to understamd the Travis docs, but I think it might work if we just remove this line... The `-name:` part on the line below creates a new build I think.

Reply:
Should be fixed now